lib/models/local_storage.dart

```diff
@@ -51,8 +51,3 @@ enum StorageKeys {
         _ => throw TypeError(),
       };
 }
-
-UserType get userType => StorageKeys.userType();
-set userType(UserType type) {
-  StorageKeys.userType.save(type.index);
-}
```

Comment:
Now that `StorageKeys` isn't a private class, it makes sense to keep `userType` and `UserType` together.